ebusd-2.1.x/de/vaillant/15.350.csv

```diff
@@ -65,7 +68,8 @@ r;wi,,eBUSCRC,,,,,"3800",,,UCH,,,
 r;wi,,HwcTempDesired,,,,,"3900",,,temp,,,
 r;wi,,ComfTempEnabled,,,,,"3A00",,,onoff,,,
 r;wi,,ComfTempDesired,,,,,"3B00",,,temp;temp;temp;temp,,,
-r;wi,,FrostProtectDelay,,,,,"3C00",,,UCH,,,
+[SW<114]r;wi,,FrostProtectDelay,,,,,"3C00",,,UCH,,,
+[SW>=114]r;w,,DesiredTemp,,,,,"3C00",,,D1C,,, # Note: Reading and writing is only effective when a specifc temperature for today has already been set on the VRT350 today. DesiredTemp is then used rather than DesiredTempLow/DesiredTempDay. How to activate DesiredTemp remotely?
```

Comment:
why don't you put this into the comment column of the field or the message? german would be preferred for the de file